spec/models/sm2_spec.rb

```diff
@@ -0,0 +1,163 @@
+      card.verify_translation("Fruit", "18.127")
+    end
+
+    context "fields are equals: easiness_factor = 2.4, number_repetitions = 0 and repetition_interval = 0" do
+      let(:card) { FactoryGirl.create :card, easiness_factor: 2.5, number_repetitions: 0,
+                                      repetition_interval: 0, review_date: DateTime.now.utc }
+      it { expect(card.easiness_factor).to eq 2.4 }
+      it { expect(card.number_repetitions).to eq 0 }
+      it { expect(card.repetition_interval).to eq 0 }
+      it { expect(card.review_date).to eq DateTime.now.utc + 12.hour }
+    end
+
+    context "fields are equals: easiness_factor = 2.4, number_repetitions = 0 and repetition_interval = 0" do
+      let(:card) { FactoryGirl.create :card, easiness_factor: 2.4, number_repetitions: 0,
+                                      repetition_interval: 0, review_date: DateTime.now.utc }
+      it { expect(card.easiness_factor).to eq 2.3 }
+      it { expect(card.number_repetitions).to eq 0 }
+      it { expect(card.repetition_interval).to eq 0 }
+      it { expect(card.review_date).to eq DateTime.now.utc + 12.hour }
+    end
+  end
+
+  describe "check correct answer and time answer > 20 seconds" do
+    before(:each) do
+      card.verify_translation("Fruit", "23.562")
+    end
+
+    context "fields are equals: easiness_factor = 2.5, number_repetitions = 0 and repetition_interval = 0" do
+      let(:card) { FactoryGirl.create :card, easiness_factor: 2.5, number_repetitions: 0,
+                                      repetition_interval: 0, review_date: DateTime.now.utc }
+      it { expect(card.easiness_factor).to eq 2.5 }
+      it { expect(card.number_repetitions).to eq 0 }
+      it { expect(card.repetition_interval).to eq 0 }
+      it { expect(card.review_date).to eq DateTime.now.utc + 12.hour }
+    end
+  end
+
+  describe "check wrong answer and 0 < time answer < =10 seconds" do
+    before(:each) do
+      card.verify_translation("Apple", "3.452")
+    end
+
+    context "fields are equals: easiness_factor = 2.5, number_repetitions = 0 and repetition_interval = 0" do
+      let(:card) { FactoryGirl.create :card, easiness_factor: 2.5, number_repetitions: 0,
+                                      repetition_interval: 0, review_date: DateTime.now.utc }
+      it { expect(card.easiness_factor).to eq 2.5 }
+      it { expect(card.number_repetitions).to eq 0 }
+      it { expect(card.repetition_interval).to eq 0 }
+      it { expect(card.review_date).to eq DateTime.now.utc + 12.hour }
+    end
+  end
+
+  describe "check wrong answer and time answer > 10 seconds" do
+    before(:each) do
+      card.verify_translation("Apple", "17.281")
+    end
+
+    context "fields are equals: easiness_factor = 2.5, number_repetitions = 0 and repetition_interval = 0" do
+      let(:card) { FactoryGirl.create :card, easiness_factor: 2.5, number_repetitions: 0,
+                                      repetition_interval: 0, review_date: DateTime.now.utc }
```

Comment:
Line is too long. [93/80]<br>Align the elements of a hash literal if they span more than one line.<br>Expression at 156, 93 should be on its own line.